cluster/aws/config-default.sh

```diff
@@ -142,6 +142,8 @@ ENABLE_NODE_PUBLIC_IP=${KUBE_ENABLE_NODE_PUBLIC_IP:-true}
 
 # OS options for minions
 KUBE_OS_DISTRIBUTION="${KUBE_OS_DISTRIBUTION:-jessie}"
+KUBE_MASTER_OS_DISTRIBUTION="${KUBE_OS_DISTRIBUTION}"
```

Comment:
I'm just curious why these wouldn't be: `KUBE_MASTER_OS_DISTRIBUTION="${KUBE_MASTER_OS_DISTRIBUTION:-$KUBE_OS_DISTRIBUTION}"`